compiler/qsc_parse/src/lib.rs

```diff
@@ -74,6 +74,9 @@ enum ErrorKind {
     #[error("missing entry in sequence")]
     #[diagnostic(code("Qsc.Parse.MissingSeqEntry"))]
     MissingSeqEntry(#[label] Span),
+    #[error("dotted namespace aliases are no longer allowed")]
```

Comment:
May be just me but I prefer this:

```suggestion
    #[error("dotted namespace aliases are not allowed")]
```

most likely, this error message will stick around forever and whoever sees it won't know these were ever allowed.